src/schema/ghga.yaml

```diff
@@ -1657,23 +1739,34 @@ slots:
     description: >-
       The type of primer used for reverse transcription, e.g. "oligo-dT" or "random" primer. 
       This allows users to identify content of the cDNA library input e.g. enriched for mRNA.
+    broad_mappings:
+      - NCIT:C1451
 
   end bias:
     description: >-
       The end of the cDNA molecule that is preferentially sequenced, 
       e.g. 3/5 prime tag or end, or the full-length transcript.
+    broad_mappings:
+      - EFO:0010187
 
   target regions:
     description: >-
-      subset of genes or specific regions of the genome, which are most likely to be involved in the phenotype under study
+      Subset of genes or specific regions of the genome, which are most likely to be involved in the phenotype under study. 
+    broad_mappings:
+      - NCIT:C130177
 
   rnaseq strandedness:
     description: >-
-      The strandedness of the library, whether reads come from both strands of the cDNA or only from the first (antisense) or the second (sense) strand.
+      The strandedness of the library, whether reads come from both strands of the cDNA 
+      or only from the first (antisense) or the second (sense) strand.
+    broad_mappings:
+      - NCIT:C48673
   
   sequencing center: 
     description: >-
       Center where sample was sequenced.
+    close_mappings:
+      - NCIT:C172274
```

Comment:
`NCIT:C172274 Sequencing Platform Name` is conceptually different from `sequencing center`. The latter is referring to an insitution/site that hosts one (or more) sequencing instruments.

`GENEPIO:0000070 sequencing location` is a better mapping.